com.dreamsourcelab.DSView.yml

```diff
@@ -0,0 +1,57 @@
+app-id: com.dreamsourcelab.DSView
+runtime: org.kde.Platform
+runtime-version: 5.15-22.08
+sdk: org.kde.Sdk
+command: DSView
+finish-args:
+  - --socket=x11
+  - --socket=wayland
+  - --share=ipc
+  - --device=all
+cleanup:
+  - /include
+  - /lib/cmake
+  - /lib/pkgconfig
+  - '*.a'
+  - '*.la'
+  - /man
+  - /share/info
+rename-icon: dsview
+rename-desktop-file: dsview.desktop
+modules:
+  - name: dsview
+    buildsystem: cmake-ninja
+    config-opts:
+      - '-DCMAKE_BUILD_TYPE=RelWithDebInfo'
+    sources:
+      - type: git
+        url: https://github.com/DreamSourceLab/DSView.git
+        tag: v1.3.0
+        commit: 02152f1c75f316b35558b494dbf22f9fb12a92bb
+      - type: patch
+        path: patches/fix_cmake_install_prefix.patch
+      - type: file
+        path: com.dreamsourcelab.DSView.metainfo.xml
+    build-commands:
+      - install -Dm644 com.dreamsourcelab.DSView.metainfo.xml -t "${FLATPAK_DEST}/share/metainfo"
+    modules:
+      - shared-modules/libusb/libusb.json
+      - name: boost
+        sources:
+          - type: archive
+            url: https://boostorg.jfrog.io/artifactory/main/release/1.83.0/source/boost_1_83_0.tar.gz
+            sha256: c0685b68dd44cc46574cce86c4e17c0f611b15e195be9848dfd0769a0a207628
+        buildsystem: simple
+        build-commands:
+          - ./bootstrap.sh --prefix=${FLATPAK_DEST}
+          - ./b2 install
+      - name: fftw
```

Comment:
fftw is in the runtime, unless you need the single precisiion version in which case, use the one in `shared-modules`